android/jni/app/organicmaps/editor/Editor.cpp

```diff
@@ -100,7 +100,15 @@ Java_app_organicmaps_editor_Editor_nativeGetMetadata(JNIEnv * env, jclass, jint
 {
   auto const metaID = static_cast<osm::MapObject::MetadataID>(id);
   ASSERT_LESS(metaID, osm::MapObject::MetadataID::FMD_COUNT, ());
-  return jni::ToJavaString(env, g_editableMapObject.GetMetadata(metaID));
+  if (osm::isSocialContactTag(metaID)) {
+    auto const value = g_editableMapObject.GetMetadata(metaID);
+    if (value.find('/') == std::string::npos) // `value` contains pagename
```

Comment:
If I understand you correctly, you propose to get rid of social network domain on PlacePage UI. This is how it will look (Android). I added one more options with leading `'/'` symbol.

![social-contacts-no-domain](https://user-images.githubusercontent.com/720808/213724371-8a6c78b8-c02a-44b7-9ee9-9a958e8e524e.png)

Which of the three options do you prefer?

Reply:
Without a slash is ok. It is not critical at all, because a better solution will be to show icons without the text. It will be obvious, and won't take much space, thus easier to use.

Reply:
We can finish with this PR, create an issue about redesigning icons in Place Page, and do it later separately.

Reply:
Ok, I did as we agreed. Place page shows only part of URL. See screenshots in PR description.